### ISSUE TITLE
Fun Times: Beat the High Score: uBlock Origin

### ISSUE THREAD

**Issue by Thorin-Oakenpants** [ghacksuserjs/ghacks-user.js#225]
I did some digging into an old post in a forum far far away ... apparently I set up my current profile with a clean FF portable 38.05 - so it's only around 2 and a bit years old (uBo and uM would have been installed same day)

Anyway, I'll start the ball rolling. 1.43 million
![ubocount](https://user-images.githubusercontent.com/16656956/29829290-1e6bbc6a-8d33-11e7-963d-c403d23d708b.png)

It's probably not that fantastic since I do not really visit that many sites with JS on, let alone spammy ad laden ones or any social media sites etc. Be interesting to see what % others have

**Comment by earthlng** [comment]
Lol, I'm not even close: 255784 or 17%

**Comment by Thorin-Oakenpants** [comment]
I expect mine to rank low, because first of all NS kills most JS to start with, so a lot of 3rd parties are never picked up on. I bet crssi is like 50 billion . he is such an internet junkie that kid.

17% - see, we have different approaches, you and me, with NS, uBo and uM. I still reckon you cause yourself more work

**Comment by Just-me-ghacks** [comment]
since install
540 or 29%

**Comment by Thorin-Oakenpants** [comment]
Sheeshus Just-me-ghacks .. are you surfing porn sites or something?

**Comment by 2glops** [comment]
1.08 million for 11% here

**Comment by GitCurious** [comment]
2,546,279 or 18%

(got onto UBO pretty soon after it appeared - Matrix was too much faffing about for me with my obsession with contant peering at the logs ;) )

**Comment by Thorin-Oakenpants** [comment]
pics or it didn't happen .. if you want the :1st_place_medal: and (secret) special prize, that is

**Comment by Thorin-Oakenpants** [comment]
Congrats @GitCurious .. :1st_place_medal: ... two rocks that keep tigers away, and a years free subscription to r/firefox are on the way

**Comment by Thorin-Oakenpants** [comment]
Well, although I didn't have to lose my stats, I did a completely clean migration to Web Extension (exported, uninstalled, manually removed traces left behind, installed, imported etc) - so I started from scratch. I'm currently at 17% for effectively 2 months stats - I expect this will go higher now I have killed off NS until it gets it shit together.

But just saw this on reddit
![reddit](https://user-images.githubusercontent.com/16656956/33244820-fdfb14b4-d362-11e7-8240-768a3e1f6be7.png)

86%?? WTF?
